frappe/utils/redis_wrapper.py

```diff
@@ -472,7 +472,7 @@ def __init__(self, maxsize: int = 1024, ttl=10 * 60, monitor: RedisWrapper | Non
 			self.invalidator_id = self.invalidator.client_id()
 		except redis.exceptions.ConnectionError:
 			# Redis not available, this can happen during setup/startup time
-			self.redis = frappe.cache
+			self.redis = frappe.local.cache
```

Comment:
```suggestion
			self.redis = frappe.cache
```

This doesn't seem correct :thinking:. It will just break if the client connection isn't working and subsequent calls try to use `get_value` on a dict.

Reply:
sorry, my bad